README.md

```diff
@@ -56,18 +56,22 @@ The following example configures the migration to only look in the `./src/main/r
 one file, `original-specification-example.foo` alone.
 ```json
 [
-  {
-    "name": "upgrade-foo-extension-files-migration",
-    "implementation": "org.technologybrewery.baton.example.FooToBarMigration",
-    "fileSets": [
-      {
-        "includes": ["**/legacy/*.foo"],
-        "excludes": ["original-specification-example.foo"]
-      }
-    ]
-  }
+    {
+        "group": "group_1",
```

Comment:
A: very minor, but recommend we make this `"group-1"` (dash vs. underscore for consistency with maven norms) or something more functionally meaningful (e..g, `"upgrade-abc-migrations"`)

Reply:
Fixed.